constants.py

```diff
@@ -822,3 +822,15 @@
 
 kPullDownBandLimit = 0.1
 """Revolutions"""
+
+kRobotToShooterTransform = Transform3d(
+    Pose3d(),
+    Pose3d(
+        -5.631 * kMetersPerInch, 0, 13.191 * kMetersPerInch, Rotation3d(0, 0, math.pi)
+    ),
+)
+
+kShooterWheelDiameter = 3.965 * kMetersPerInch
```

Comment:
whats this wheel diameter from? I though they were 4in

Reply:
cad

Reply:
![image](https://github.com/1757WestwoodRobotics/2024-Crescendo/assets/91603936/2514d135-0b33-4b43-8a94-84314d7d937e)
:)

Reply:
ur gaslighting me

Reply:
the andymark wheels in cad have a bit of a ramp on the sides, idk how accurate it is but that number isn't from the center wheel radius
treat it as a 4in wheel cuz the ttb wheels are 4in